src/Core/FormatFactorySettings.h

```diff
@@ -1311,6 +1311,27 @@ Set the quoting rule for identifiers in SHOW CREATE query
 )", 0) \
     DECLARE(IdentifierQuotingStyle, show_create_query_identifier_quoting_style, IdentifierQuotingStyle::Backticks, R"(
 Set the quoting style for identifiers in SHOW CREATE query
+)", 0) \
+    DECLARE(String, output_png_image_pixel_format, "RGB", R"(
```

Comment:
We name these settings `output_format_...` everywhere, so here is should be `output_format_png_...`, not `output_png_...`